iree/compiler/Codegen/Passes.h

```diff
@@ -254,53 +254,51 @@ createLLVMGPUDistributeSharedMemoryCopy();
 std::unique_ptr<OperationPass<FuncOp>> createLLVMGPUPipeliningPass();
 
 //------------------------------------------------------------------------------
-// SPIRV Passes
+// SPIR-V Passes
 //------------------------------------------------------------------------------
 
-/// Pass pipeline to lower executable obtained from Linalg tile + distribute to
-/// scalar + vector code. Does distribution to threads (no vectorization).
-void addSPIRVDistributePassPipeline(OpPassManager &pm);
+/// Pass pipeline to lower IREE HAL executables with workgroup tiled and
+/// distributed Linalg ops to SPIR-V scalar code. Additionally performs
+/// distribution to threads without vectorization.
+void addSPIRVTileAndDistributePassPipeline(OpPassManager &pm);
 
-/// Pass pipeline to lower executables that contain operations that are not
-/// tiled + distributed.
-void addSPIRVDistributeToGlobalIDPipeline(OpPassManager &pm);
+/// Pass pipeline to lower IREE HAL executables with not tiled/distributed
+/// Linalg ops. Performs distribution to global invocations.
+void addSPIRVDistributeToGlobalIDPassPipeline(OpPassManager &pm);
 
-/// pipeline to lower executable obtained from Linalg tile + distribute to
-/// scalar + vector code. Does distribution to threads and vectorization.
-void addSPIRVVectorizationPassPipeline(OpPassManager &pm);
+/// Pass pipeline to lower IREE HAL executables with workgroup tiled and
+/// distributed Linalg ops to SPIR-V scalar and vector code. Additionally
+/// performs distribution to threads with vectorization.
```

Comment:
Nit : Doesnt necessarily distribute to threads. It could distribute to workgroups right?

Reply:
That's mentioned earlier with `... workgroup tiled and distributed...`